src/NuGet.Core/NuGet.Packaging/Signing/Timestamp/Rfc3161TimestampTokenInfoNetstandard21Wrapper.cs

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using System.Security.Cryptography;
+
+namespace NuGet.Packaging.Signing
+{
+#if IS_SIGNING_SUPPORTED && IS_CORECLR
+    internal sealed class Rfc3161TimestampTokenInfoNetstandard21Wrapper : IRfc3161TimestampTokenInfo
```

Comment:
Do we need this? We are only using net5 aren't we?

Reply:
Or maybe we do need and should name it so something better :) 

Versions change :) 
Future Pr would be ok.

Reply:
I think the logic of `#if IS_SIGNING_SUPPORTED && IS_CORECLR` is more clear than` #if NET5_0`.
We can tell the first one, is related to signing, and when it's the netcore code path.
The second one may be related to signing, or may be related to net5.0 API change unrelated to signing.